File: CS102/CS102-DZ08/src/cs102/dz08/Student.java
package cs102.dz08;

public class Student {

    private String ime;
    private String prezime;
    private String index;

    public Student() {
    }

    public Student(String ime, String prezime, String index) {
        this.ime = ime;
        this.prezime = prezime;
        this.index = index;
    }

    public String getIme() {
        return ime;
    }

    public void setIme(String ime) {
        this.ime = ime;
    }

    public String getPrezime() {
        return prezime;
    }

    public void setPrezime(String prezime) {
        this.prezime = prezime;
    }

    public String getIndex() {
        return index;
    }

    public void setIndex(String index) {
        this.index = index;
    }

    @Override
    public String toString() {
        return "Student{" + "ime=" + ime + ", prezime=" + prezime + ", index=" + index + '}';
    }
}

File: CS323/CS323-DZ11/CS323-DZ11/Student.cpp
#include "Student.h"


Student::Student(int index) {
	this->index = index;
	}

Student::Student() {

	}

Student::~Student() {
	cout << "Student je unisten!" << endl;
	}

void Student::setIndex(int index) {
	this->index = index;

	}

int Student::getIndex() {
	return index;
	}

string Student::toString() {
	string str = ime;
	str += " " + prezime;
	str += " " + adresa;
	str += " " + telefon;
	str += " " + index;
	return str;
}
